sirepo/auth_db.py

```diff
@@ -64,12 +64,9 @@ def audit_proprietary_lib_files(uid):
             sirepo.auth.role_for_sim_type(t),
         )
         d = sirepo.srdb.proprietary_code_dir(t)
-        assert d.exists(), \
-            f'{d} proprietary_code_dir must exist' \
-            + ('; run: sirepo setup_dev' if pykern.pkconfig.channel_in('dev') else '')
-
+        if not d.exists():
```

Comment:
we need configuration for this, perhaps. or at least do this check for flash or codes that need it. this can be hardwired in the sim_data so that we can ask the question of the code.

Reply:
4c19d124